src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/Processor.cs

```diff
@@ -371,15 +371,10 @@ internal XsltOutput Output
 
             _scriptExtensions = new Hashtable(_stylesheet.ScriptObjectTypes.Count);
             {
-                foreach (DictionaryEntry entry in _stylesheet.ScriptObjectTypes)
+                // Scripts are not supported on stylesheets
```

Comment:
Do we need a test for this or something? Are we concerned about the behavior change?

Reply:
This is a functional change as far as I can tell - we should have the proper tests and get it reviewed by the area owners.

Reply:
Agreed, @krwq AFAICT Script objects on xsls are not supported at all in .NET Core. I tried with several different approaches trying to create an xsl that had scripts and all entrypoints that I found trhough XslCompiledTransform would all eventually throw PNSE. I think that this has been dead code that we just originally imported from .NET Framework, but never really did anything as it throws PNSE when you actually invoke the transform operation.

Reply:
IIRC that's correct and intentional. Might be a good idea to remove this code completely and just shortcut to PNSE to save some bytes.